packages/vitest/src/defaults.ts

```diff
@@ -43,6 +43,8 @@ const defaultCoverageExcludes = [
   '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*',
   '**/vitest.{workspace,projects}.[jt]s?(on)',
   '**/.{eslint,mocha,prettier}rc.{?(c|m)js,yml}',
+  ...defaultExclude,
+  ...defaultInclude,
```

Comment:
Shouldn't it include user defined include/exclude? Somewhere here: https://github.com/vitest-dev/vitest/blob/d7f23d08c4a5420bb0b6483c7e3dc87a1ae8eeaf/packages/vitest/src/node/config.ts#L510

Reply:
I'm not sure about that. I would rather keep `test.include` and `test.coverage.include` runtime values separately. Less automation to make things simple. I don't think Jest that does that either. 

Though I don't have strong opinion about this.

Reply:
I mean what is the reason to merge them then?

Reply:
Current coverage default excludes seem to include most of the patterns already. Let's just remove the merging here.

Reply:
But overall: I would like to avoid situation where configuring `test.inlcude/exclude` magically affects `test.coverage.include/exclude` in cases where user doesn't configure `test.coverage.include/exclude` in the first place.

Reply:
Yeah I am fine with removing it or adding the configured include/exclude only to non-configured `coverage.exclude`, but before we make a decision we need to find the reason why we even had that at the first place. I think it makes sense to remove any test file from coverage by default, don't you?

Reply:
Sure, we could do that. It's not really related to https://github.com/vitest-dev/vitest/issues/5965 though, as that one is only about being able to overwrite the defaults.

If we automatically added user's `test.include/exclude´ to `test.coverage.include/exclude´, would it then be impossible to fully overwrite `test.coverage.exclude´? Or would we only apply `test.include/exclude´ there if coverage ones were not set by user?

Reply:
The last one. Only apply if there is no exclude already

Reply:
> Somewhere here:
> 
> [vitest/packages/vitest/src/node/config.ts](https://github.com/vitest-dev/vitest/blob/d7f23d08c4a5420bb0b6483c7e3dc87a1ae8eeaf/packages/vitest/src/node/config.ts#L510)

At this point the default values for coverage have already been applied, probably by `src/node/plugins/index.ts`.

Reply:
Can we do it in plugins/index then? 😄

Reply:
Added it into plugins. There's already some configuration resolving so it's not that bad to add some more 😄